### ISSUE TITLE
make a detail description for running this project?

### ISSUE THREAD

**Issue by fangjj** [Monsquaz/website-backend#1]
can you make a detail description for running this project？ I git clone this project，but i don't know how to use the seeds and migrations files. I config the config.js and knexfile.js and run the command knex seed:run and knex migrate:latest, but get a error！so, i wish you can help me out!

**Comment by RAMPKORV** [comment]
I wasn't anticipating that someone would want to run it this early :1st_place_medal: . The project is still a one-man-show in early in development, and the plan was to keep it so for another month or so. I'll get write some details shortly. What error(s) do you get, though?

**Comment by fangjj** [comment]
thank for your reply! I found the project by searching the keywords "join-monster mysql" on github. I want to  learn lessons from your project using join-monster and mysql. I run the project and create a empty database in mysql, and config it in knexfile.js. then, i run knex migrate:latest command, got this ...
<img width="815" alt="wx20170830-211101 2x" src="https://user-images.githubusercontent.com/15318800/29874084-3ff796fc-8dc8-11e7-84a6-5f43958bae23.png">
I can't fix it by reading the knex doc, but got nothing. so can you help me?

**Comment by RAMPKORV** [comment]
Ahh! That. 

You need `require('babel-register')` in your knexfile so we can use es6 syntax in migrations and seeds.
I pushed an update to the knexfile.js.example now (should've done earlier).

**Comment by fangjj** [comment]
yep, it doesn't work! <img width="1095" alt="wx20170830-230222 2x" src="https://user-images.githubusercontent.com/15318800/29879437-9fce5386-8dd7-11e7-98ca-14e1117e53e5.png">
why not use sql script to create table and data?

**Comment by RAMPKORV** [comment]
If `knex migrate:latest` doesn't work, then `knex seed:run` won't, because the migrations create all the tables. From the looks of the screenshot, it still seems like the migration command didn't go through, and it looks to be an ES6 issue.

I'll have to investigate this, and also try making a clean install again to make sure it works, and document the requirements.

I decided to write the database through migrations from start as a learning experience, mainly, since the project is not put in production and won't be for a few months. From then on, however, writing database changes as migrations will be necessary.
